sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/AggregationQuerySuite.scala

```diff
@@ -545,19 +576,21 @@ abstract class AggregationQuerySuite extends QueryTest with SQLTestUtils with Te
           |  count(distinct value2),
           |  sum(distinct value2),
           |  count(distinct value1, value2),
+          |  longProductSum(distinct value1, value2),
```

Comment:
Actually, what is the semantic of this? If we have `[value1=null, value2=null], [value1=null, value2=1], [value1=1, value2=null], and [value1=1, value2=1]`, we have 4 distinct input rows, right?